benchmark/compare.js

```diff
@@ -3,6 +3,7 @@
 const fork = require('child_process').fork;
 const path = require('path');
 const CLI = require('./_cli.js');
+const BenchmarkProgress = require('./_benchmark_progress');
```

Comment:
nit: I personally prefer to be explicit with `.js`, but I don't think we have a rule.